src/main/java/com/acme/rx/RxBroadcast.java

```diff
@@ -21,6 +21,7 @@ private RxBroadcast() {
         public void handle(Intent intent) {
           subscriber.onNext(new Intent());
           proxyConsumer.accept(new BroadcastReceiverProxy(this));
+          throw new RuntimeException("Intentional exception");
```

Comment:
The assumption we should make is that the library code works. I'm not sure why you are throwing exception in the library code.

The exception comes from the user's code.